docs/source/a_quick_tour.mdx

```diff
@@ -200,6 +199,47 @@ The `combine` function accepts both the list of names of the metrics as well as
 }
 ```
 
+## Running evaluation on a suite of tasks
+
+It can be useful to evaluate models on a variety of different tasks to understand their downstream performance. The [EvaluationSuite](evaluation_suite) enables evaluation of models on a collection of tasks. Tasks can be constructed as ([evaluator](base_evaluator), dataset, metric) tuples and passed to an [EvaluationSuite](evaluation_suite) stored on the Hugging Face Hub as a Space. See the [evaluator documentation](base_evaluator) for a list of currently supported tasks.
+
+`EvaluationSuite` scripts can be defined as follows, and supports Python code for data preprocessing.
+
+```python
+import evaluate
+from evaluate.evaluation_suite import SubTask
+
+class Suite(evaluate.EvaluationSuite):
+
+    def __init__(self, name):
+        super().__init__(name)
+        self.preprocessor = lambda x: {"text": x["text"].lower()}
```

Comment:
the preprocessor would also need to be added to `Subtask.data_preprocessor`, right? I think we could stick to the absolute minimum here and omit preprocessing. With only one task it might be not so clear what the upside vs. using the evaluator is. What do you think about the following example:
- use IMDb and SST-2
- no need for `split`, `subset`, `metric` in the example here

What's a bit confusing in the example as it is, is that the code you show doesn't match the evaluation suite that you later run. In the dedicated docs page we could then use the more advanced example of glue. What do you think?

Reply:
I'll swap it all to IMDB + SST2 for simplicity and get rid of the preprocessor!

I think we should keep `split` because otherwise the example will take too long to run, and I believe `metric` is necessary because otherwise the evaluator won't run, but I'll drop `split`.

Reply:
sounds good, I guess you meant `subset` at the end :)